sdk/mongocluster/arm-mongocluster/README.md

```diff
@@ -0,0 +1,57 @@
+# Azure MongoClusterManagement REST client library for JavaScript
```

Comment:
```suggestion
# Azure MongoClusterManagement client library for JavaScript
```
This should not include REST here.

Reply:
update manually and create an issue here https://github.com/Azure/autorest.typescript/issues/2637